package.json

```diff
@@ -85,7 +85,7 @@
     "babel": "rimraf ./lib && babel src --out-dir lib --copy-files",
     "coveralls": "nyc report --reporter=text-lcov | coveralls",
     "dev": "rimraf ./lib && babel -w src --out-dir lib --copy-files",
-    "lint": "eslint '{src,test}/**/*.js'",
+    "lint": "eslint src/**/*.js test/**/*.js",
```

Comment:
Probably some quoting is needed to prevent shell from expanding those wildcards on some platforms

Reply:
Yes you are right. Updated this to use double quotes - which prevent globbing in sh. Single quotes are problematic in cmd.